commonui/src/main/java/com/hyperwallet/android/ui/common/util/CurrencyParser.java

```diff
@@ -127,4 +127,29 @@ public CurrencyDetails getCurrency(String currencyCode) {
         }
         return null;
     }
+
+    /**
+     * truncate decimals for given value
+     *
+     * @param value Any value in string.
+     * @param noOfDecimals   number of decimal to be truncate.
+     * @return Returns truncated decimal value.
+     */
+    public static String getValueWithTruncateDecimals(String value, int noOfDecimals) {
+        if (value != null) {
+            StringBuilder builder = new StringBuilder();
+            String[] amount = value.split("\\.");
+            if (amount.length == 2) {
+                String wholeNumber = amount[0];
+                String decimal = amount[1];
+                if (decimal.length() > noOfDecimals) {
+                    decimal = decimal.substring(0, noOfDecimals);
+                }
+                return builder.append(wholeNumber).append(".").append(decimal).toString();
+            }
```

Comment:
why make it so complicated? Please change the whole code with 

String str = String.substring(0, value.indexOf("\\.") + noOfDecimals)

Reply:
updated

Reply:
How do I do that and what does this mean